bigchaindb/commands/bigchain.py

```diff
@@ -122,7 +122,7 @@ def run_configure(args, skip_if_exists=False):
 def run_export_my_pubkey(args):
     """Export this node's public key to standard output
     """
-    logger.debug('bigchaindb args = {}'.format(args))
+    print('bigchaindb args = {}'.format(args))
```

Comment:
Mmm, maybe this should stay as a logging statement (and just moved somewhere else?)